### PR TITLE
Route to appropriate tab when search is filtered.

### PR DESCRIPTION
**Pull request by sri2k1us**
- SSIA

### CODE REVIEW

File: src/components/search/GlobalSearchField.js [hunk 1]
```diff
@@ -649,9 +649,19 @@ function GlobalSearchField(props) {
             onKeyDown={(e) => {
                 if (e.key === "Enter") {
                     setOpen(false);
-                    router.push(
-                        `/${NavigationConstants.SEARCH}?searchTerm=${searchTerm}&filter=${filter}`
-                    );
+                    if (filter !== searchConstants.ALL) {
```
Comment (ash3rz) [inline src/components/search/GlobalSearchField.js:652]: There's another check below to see if the filter is set to `ALL`.  I think we still want users to be directed to the search page when they hit the `Enter` key, so perhaps this check is unnecessary?

**Review by ash3rz — APPROVED** [review]
Had a small question, otherwise LGTM 👍

File: src/components/search/GlobalSearchField.js (hunk 1 above)
Comment (sri2k1us) [reply src/components/search/GlobalSearchField.js]: Good Catch. Indeed a bug. Should be fixed now.

**Review by psarando — APPROVED** [review]
LGTM :shipit: 👍

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**